Renderer/CKEditorRenderer.php

```diff
@@ -13,8 +13,8 @@
 
 use Ivory\JsonBuilder\JsonBuilder;
 use Symfony\Component\Asset\Packages;
-use Symfony\Component\DependencyInjection\ContainerInterface;
 use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RequestStack;
 use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
 use Symfony\Component\Routing\RouterInterface;
 use Symfony\Component\Templating\EngineInterface;
```

Comment:
IIRC you should depend on twig directly instead of templating.

Reply:
1. Twig is optional.
2. IMO we should let the `FrameworkBundle` handle which templating engine is used. That way we do not care whether it is Twig or Symfony's templating.

Reply:
Did not see it was optional, my bad.